tdp_core/security/manager/jwt_security_manager.py

```diff
@@ -1,70 +1,38 @@
+import logging
+from abc import ABC
+from base64 import b64decode
+from datetime import datetime, timedelta, timezone
 from functools import wraps
-from typing import Dict, List, Optional, Union
-from flask import request, abort, jsonify, render_template_string, Flask, Request
-from phovea_server.security import SecurityManager as PhoveaServerSecurityManager, User as PhoveaServerUser
-from phovea_server.plugin import list as list_plugins
-from phovea_server.ns import no_cache
+from typing import Dict, Optional
+
+from flask import request, abort, jsonify, render_template_string, Flask, Request, Blueprint, current_app, _request_ctx_stack
```

Comment:
Some flake8 issues remain due to imports: https://app.circleci.com/pipelines/github/datavisyn/tdp_core/1413/workflows/ec003492-643c-41bc-b3df-0945f273751f/jobs/4133